components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.test.tsx

```diff
@@ -132,4 +135,67 @@ describe('components/admin_console/permission_schemes_settings/permission_tree',
         expect(groups[8].id).toStrictEqual('manage_shared_channels');
         expect(groups[9].id).toStrictEqual('custom_groups');
     });
+
+    describe('should show playbook permissions', () => {
+        it('for starter license', () => {
+            const props = {
+                ...defaultProps,
+                license: {
+                    isLicensed: 'true',
+                    SkuShortName: LicenseSkus.Starter,
+                },
+            };
+
+            const wrapper = shallow(
+                <PermissionsTree
+                    {...props}
+                />,
+            );
+
+            const groups = wrapper.find(PermissionGroup).first().prop('permissions');
+            expect(groups[3].id).toStrictEqual('playbook_public');
+            expect(groups[4].id).toStrictEqual('runs');
+        });
```

Comment:
Is this behavior for Starter level only for cloud? Cloud is the environment where an actual Starter license is used. In self-managed, Starter plan is equivalent to no license (EE or TE). Just wondering if we need to include that condition in this PR

Reply:
Yes, would be good to cover all the cases. We can probably group TE and E0 in the `isLicensed: 'false'` case.